tools/GenerateAppxFromManifest.ps1

```diff
@@ -0,0 +1,74 @@
+# This script is used for taking all the activatable classes from a SxS manifest
+# and adding them as Extensions to an Appxmanifest.xml.
+# Params:
+#  - SxSManifest: The path to the SxS manifest to get the types from
+#  - AppxManifestPrototype: The path to an AppxManifest.xml-style XML document to add the Extensions to
+#  - SxSManifest: The path to write the updated XML doc to.
+
+param (
+    [parameter(Mandatory=$true, Position=0)]

```

Comment:
When calling this script, please use named parameters: `-SxSManifest whatever.xml -AppxManifestPrototype whatever2.xml`